CleverFerret/src/main/java/com/universalmedialibrary/services/music/AudioEffectsController.kt

```
@@ -0,0 +1,161 @@
+        } catch (e: Exception) {
+            // Audio effects not available on this device
+        }
+    }
+
+    fun enableReverb(enabled: Boolean) {
+        try {
+            currentReverbEnabled = enabled
+            audioEffectsService.setReverb(currentReverbPreset, enabled)
+            persist { putBoolean(KEY_REVERB_ENABLED, enabled) }
+        } catch (e: Exception) {
+        }
```

detekt (1.23.8)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)